frontend/src/app/features/work-packages/components/wp-form-group/wp-attribute-group.component.sass

```diff
@@ -1,8 +1,47 @@
 @import "src/assets/sass/helpers"
 
 .wp-attribute-group
+  display: flex
```

Comment:
Wouldn't it be better to use a `grid` here?  Thus we could avoud all these magic calculations

Reply:
Unfotunately no, because of the requirement that the the fields go single column at some point, but we cannot use breakpoints since the width of the layout is independent of the width of the screen. You can show/hide and drag the sidemenu and split screen.